resources/windows/wix/customactions.wxs.erb

```diff
@@ -50,6 +50,29 @@ End If
       ]]>
     </CustomAction>
 
+    <!-- Custom action to get the localised name for the "Everyone" (or World) user as
+         WixUtil:OSInfo Custom action doesn't return this value.
+         Equivalent of     (Get-WmiObject Win32_Account | where { $_.SID -eq "S-1-1-0" }).name
+      -->
+    <CustomAction
+      Id="SetEveryoneNameFromSid"
```

Comment:
Minor - can we change the name of the CustomAction to something more generic as it gets SIDs for Admins and Everyone now?